dependency-check-suppressions.xml

```diff
@@ -52,6 +52,15 @@
    ]]></notes>
 		<cve>CVE-2016-1000027</cve>
 	</suppress>
+
+	<suppress until="2022-05-25">
+		<notes>
+			CVE affecting spring-framework.version 5.3.20
+			Currently temporary suppression to fix failing pipeline build
+		</notes>
+		<cve>CVE-2016-1000027</cve>
```

Comment:
Remove this duplicate suppression as the same CVE was suppressed at line 53.

Reply:
Done